apps/src/gamelab/GameLabSprite.js

```diff
@@ -575,159 +578,7 @@ var AABBops = function (p5Inst, type, target, callback) {
-                                 (this.position.y+this.previousPosition.y)/2);
-
-            //the extents are the distance between the coll centers
-            //plus the extents of both
-            var e = createVector(
-                                 abs(this.position.x -this.previousPosition.x) + this.collider.extents.x,
-                                 abs(this.position.y -this.previousPosition.y) + this.collider.extents.y);
-
-            var bbox = new AABB(p5Inst, c, e, this.collider.offset);
-
-            //bbox.draw();
-
-            if(bbox.overlap(other.collider))
-            {
-              if(tunnelX) {
-
-                //entering from the right
-                if(this.velocity.x < 0)
-                  displacement.x = other.collider.right() - this.collider.left() + 1;
-                else if(this.velocity.x > 0 )
-                  displacement.x = other.collider.left() - this.collider.right() -1;
-              }
-
-              if(tunnelY) {
-                //from top
-                if(this.velocity.y > 0)
-                  displacement.y = other.collider.top() - this.collider.bottom() - 1;
-                else if(this.velocity.y < 0 )
-                  displacement.y = other.collider.bottom() - this.collider.top() + 1;
-
-              }
-
-            }//end overlap
-
-          }
-          else //non tunnel overlap
-          {
-            /* First, check for adjacency using overlap method.
-             *
-             * We do this to ensure isTouching returns true for cases when
-             * bounce, collide would return true and also when overlap would
-             * return true.
-             *
-             * NOTE: this.touching will remain all false in the adjacent case,
-             * but the function will return true.
-             */
-
-            //if the other is a circle I calculate the displacement from here
-            //and reverse it
-            if (this.collider instanceof CircleCollider) {
-              if (other.collider.overlap(this.collider)) {
-                result = true;
-                displacement = other.collider.collide(this.collider).mult(-1);
-              }
-            }
-            else if (this.collider.overlap(other.collider)) {
-              result = true;
-              displacement = this.collider.collide(other.collider);
-            }
-          }
-
-          if(displacement.x !== 0 || displacement.y !== 0)
-          {
-            if(displacement.x > 0)
-              this.touching.left = true;
-            if(displacement.x < 0)
-              this.touching.right = true;
-            if(displacement.y < 0)
-              this.touching.bottom = true;
-            if(displacement.y > 0)
-              this.touching.top = true;
-          }
-        }//end collider exists
-      }//end this !== others[i] && !this.removed
-
-  return result;
+  return this.AABBops('collide', target, function() {}, false);
 };
 
 /* eslint-enable */
```

Comment:
I know this is tangential, but will you move this `/* eslint-enable */` marker above our `isTouching` implementation?  We just disable linting because the AABBops code copied over from p5.play is messy - now that we're not copying a bunch of it into isTouching let's at least lint that method.